.gitignore

```diff
@@ -160,3 +160,10 @@ openai/
 
 # news
 CURRENT_BULLETIN.md
+
+# AgBenchmark
+agbenchmark/reports/
+
+# Nodejs
+package-lock.json
+package.json
```

Comment:
Still not sure if this was intentional or not :)

Reply:
Dunno if it was because of some of the other agents being benchmarked, but I ended up with some Nodejs files. So I added this to prevent them from being pushed to the repo.